app/views/admin/loans/questionnaires/_questionnaire_group.html.slim

```diff
@@ -84,12 +84,14 @@
               = f.input_field :"#{question.attribute_sym}[number]", as: :decimal,
                 value: response.number
 
-            - tags = %w(table tbody tr td b i u ol ul li p)
+            - tags = %w(table tbody tr td b i u ol ul p img)
```

Comment:
We need to add `li` here.

Reply:
the lists were not saving appropriately

unordered and ordered @emitche

Reply:
Oh. So we need to not include `li`?

Reply:
I've tested the current functionality on staging. v2.11.1 uses this line: `- tags = %w(table tbody tr td b i u ol ul li p)` So we do need to include `li` in this list.

I will make that change.